packages/apple-pay-integration/src/apple-pay-customer-strategy.ts

```diff
@@ -473,9 +473,12 @@ export default class ApplePayCustomerStrategy implements CustomerStrategy {
                 );
             }
 
+            await this._paymentIntegrationService.verifyCheckoutSpamProtection();
```

Comment:
What would it look like when visible captcha is triggered?

Reply:
So far in my testing, I haven't seen the visible captcha being triggered 🤔 Is there a way we can force trigger it?

Reply:
Checked offline and tested with user agent, once the challenge is visible. It completely breaks apple pay sheet and requires safari to be force quit. Hence will need to try another approach.